Twenty403/src/main/java/org/firstinspires/ftc/twenty403/command/autonomous/AutoConstants.java

```diff
@@ -48,112 +49,32 @@ public static class Right {
         public static ConfigurablePose TERMINAL = new ConfigurablePose(61, -64, toRadians(180));
         // These are 'trajectory pieces' which should be named like this:
         // {STARTING_POSITION}_TO_{ENDING_POSITION}
-        public static final Function<Function<Pose2d, TrajectorySequenceBuilder>, TrajectorySequence>
-                TELESTART_TO_FORWARD_MOVE =
-                b -> b.apply(TELESTART.toPose())
-                        .lineToLinearHeading(FORWARD_MOVE.toPose())
-                        .build(),
-                TELESTART_TO_BACKWARD_MOVE =
-                        b -> b.apply(TELESTART.toPose())
-                                .lineToLinearHeading(BACKWARD_MOVE.toPose())
-                                .build(),
-                TELESTART_TO_LEFT_MOVE =
-                        b -> b.apply(TELESTART.toPose())
-                                .lineToLinearHeading(LEFT_MOVE.toPose())
-                                .build(),
-                TELESTART_TO_RIGHT_MOVE =
-                        b -> b.apply(TELESTART.toPose())
-                                .lineToLinearHeading(RIGHT_MOVE.toPose())
-                                .build(),
+        public static final TrajectoryPath
+                TELESTART_TO_FORWARD_MOVE = TrajectoryPath.lineToLinearHeading(TELESTART, FORWARD_MOVE),
```

Comment:
This looks a lot cleaner

Reply:
It's definitely cleaner, but it's less flexible. I need to make sure that there are examples that use the TrajectoryBuilder directly, as you can string multiple trajectories together (though it's probably all gotta change if/when we eventually migrate to RoadRunner 1.0)